zbus/src/blocking/proxy/mod.rs

```diff
@@ -470,11 +470,11 @@ where
     T: TryFrom<zvariant::OwnedValue>,
     T::Error: Into<crate::Error>,
 {
-    // Get the value of the property that changed.
-    //
-    // If the notification signal contained the new value, it has been cached already and this call
-    // will return that value. Otherwise (i-e invalidated property), a D-Bus call is made to fetch
```

Comment:
Hmm.. I always wrote it "i-e". My whole life has been a life. :open_mouth:

Reply:
I mean I wouldn't go so far as to call it _wrong_, but `i.e.` is certainly more traditional 😅